app/assets/stylesheets/admin/nonprofit_applications/table.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Styling for the nonprofit application list in the admin view.
+ *
```

Comment:
I'm gonna ignore this because I'm deliberately putting the empty rule there to show the structure of the HTML.